msteams-platform/concepts/deploy-and-publish/appsource/prepare/review-copilot-validation-guidelines.md

```diff
@@ -289,7 +264,7 @@ For a Copilot agent to be validated, invoked, and to work seamlessly, ensure tha
 |---|---|
 | Manifest version | *App manifest version must be 1.13 or later. [*Must fix*] <br>* If you're using declarative agent, you must use [public developer preview app manifest schema](../../../../resources/schema/manifest-schema-dev-preview.md). [*Must fix*] |
 | Response time | Response time mustn't exceed nine seconds for 99 percent, five seconds for 75 percent and two seconds for 50 percent. [*Must fix*] |
-| Reliability | Apps must maintain 99.9% availability. For instance, if Microsoft 365 Copilot calls a plugin 1,000 times, it must provide a meaningful response 999 times. [*Must fix*] |
+| Reliability | Apps must maintain 99.9% availability. For instance, if Microsoft 365 Copilot calls a agent 1,000 times, it must provide a meaningful response 999 times. [*Must fix*] |
```

Comment:
Another one here: *an agent* . . . try doing a find/replace for "a agent"

Reply:
updated as suggested